### ISSUE TITLE
IIB Docker Image: Configuring Broker while creating the Image

### ISSUE THREAD

**Issue by kaaviraaj** [ot4i/iib-docker#51]
IBM IIB image comes with only few instructions.
Create Integration node, if it doesn't exist
Create Integration servers
Deploy bar file.

Did anyone tried Configuring a full functional IIB while creating image
Operations such as
Configuring SSL
ODBC Setup
LDAP Authentication for WebGUI?

I was trying to use IBM provided Image as base and tried to implement this, but while executing the steps I am getting below error. 
"mqsilist: not found"

It is not recofnising any mqsi commands.

Any pointers?

Thanks

**Comment by dennisseidel** [comment]
Yes I did you can find my last version under https://github.com/denseidel/ibm-integration-bus-docker-bestpractice. I hope this might help you. Disclaimer: I no longer work with IIB, so the code is not maintained anymore and I can not provide any support on it.
